addons/l10n_fr_certification/views/account_sale_closure.xml

```diff
@@ -0,0 +1,77 @@
+                <field name="total_interval"/>
+                <field name="total_beginning"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="form_view_account_sale_closure" model="ir.ui.view">
+        <field name="name">Sales Closings</field>
+        <field name="model">account.sale.closure</field>
+        <field name="arch" type="xml">
+            <form create="false" edit="false" string="Account Closure">
+                <sheet>
+                        <div class="oe_title">
+                            <h1>
+                                <field name="name"/>
+                            </h1>
+                        </div>
+                    <group>
+                    <group>
+                        <group>
+                            <field name="date_closure_start"/>
+                            <field name="date_closure_stop"/>
+                            <field name="frequency"/>
+                            <field name="sequence_number" groups="base.group_no_one"/>
+                        </group>
+                        <group>
+                            <field name="company_id" groups="base.group_multi_company"/>
+                            <field name="currency_id" invisible="1"/>
+                        </group>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="total_interval"/>
+                            <field name="total_beginning"/>
+                        </group>
+                    </group>
+                </group>
+                <notebook>
+                    <page string="Journal Entries" groups="account.group_account_user">
+                        <field name="move_ids" groups="account.group_account_user"/>
+                    </page>
+                </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_list_view_account_sale_closure" model="ir.actions.act_window">
+        <field name="name">Sales Closings</field>
+        <field name="res_model">account.sale.closure</field>
+        <field name="view_type">tree</field>
+        <field name="view_mode">list,form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_nocreate">
+                The closings are created by Odoo
+                </p><p>
+                Sales closings run automatically on a daily, monthly and annual basis. It computes both period and cumulative totals from all the sales entries posted in the system after the previous closing.
+            </p>
+        </field>
+    </record>
+
+    <menuitem action="action_list_view_account_sale_closure" id="menu_account_closure" parent="inalterability_fr_statements_menu" sequence="90"/>
+
+    <menuitem action="action_list_view_account_sale_closure" id="menu_account_closure_reporting" parent="l10n_fr.account_reports_fr_statements_menu" sequence="90"/>
```

Comment:
again, why twice the menutiems?? choose and keep only one, I say.

Reply:
That was a ProductOwner request

Reply:
@fgi-odoo choose one, dismiss the other.

Reply:
until < 11.0 I guess the choice can be made, but in 11.0, account is independant from POS in terms of views at least, so no choice here